cardano-api/src/Cardano/API.hs

```diff
@@ -65,7 +65,10 @@ module Cardano.API (
     -- ** Shelley addresses
     makeShelleyAddress,
     PaymentCredential(..),
+    fromShelleyPaymentCredential,
+    fromShelleyStakeCredential,
     StakeAddressReference(..),
+    fromShelleyStakeReference,
```

Comment:
w.r.t. these exports you've added, if you need them, could we export them from `Cardano.Api.Shelley` instead?

Reply:
Yes, these should _only_ be exported from `Cardano.Api.Shelley` and not from `Cardano.Api`

Reply:
Done.